pyccel/ast/core.py

```diff
@@ -2747,7 +2747,9 @@ class Import(Basic):
 
     Parameters
     ----------
-    target : str, list, tuple
+    source : str, DottedName, AsName
+        the module from which we import
+    target : str, AsName, list, tuple
         targets to import
```

Comment:
Thanks for updating this.

I am now convinced that the parameters' names are quite unfortunate, and that confuses me a lot. The Python [documentation](https://docs.python.org/3/reference/simple_stmts.html#the-import-statement) does not talk about "sources" and "targets", but rather "modules" and "identifiers". In fact the example

```
Import('foo')
```
Has a source but not a target, hence according to our docstring this means that we are importing nothing from module `foo`, which is not the case of course.

Even semantically, the terms "source" and "target" makes no sense to me here, because the target belongs to the source (the imported name belongs to a module).

For this class specifically, I think that we should change the parameters to `module` and `names`, and the method `define_target` should be called `add_imported_name`. In class `PythonCodePrinter`, the method `insert_new_import` should have parameters `(module, name, alias)`.


Do we agree with me? We can do this in another PR of course.

Reply:
> Has a source but not a target, hence according to our docstring this means that we are importing nothing from module foo, which is not the case of course.

I think this actually is the case when translating to fortran, if there's no target then we drop the import

> For this class specifically, I think that we should change the parameters to `module` and `names`

This makes sense to me but I think it's better kept for another PR